build.gradle

```diff
@@ -294,8 +313,6 @@ repositories {
 }
 
 dependencies {
-    implementation project(":calcite-adapter")
-    implementation project(":common")
```

Comment:
this code looks good, it's just that I see calcite-adaptor and common folders are still there despite most of their code is gone, should those two folders be deleted?

Reply:
Interesting, I deleted it here. I will double-check

Reply:
yea the folders are there on my end, but when I delete them, Git cannot commit that change, which is weird. I think the folders are safely deleted on your branch, it is just that java is weird with Git when it comes to folders